src/NHibernate/SchemaValidationException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace NHibernate
+{
+	public class SchemaValidationException : HibernateException
+	{
+		public ReadOnlyCollection<string> ValidationErrors { get; }
+
+		public SchemaValidationException(string msg, IList<string> validationErrors) : base(msg)
```

Comment:
`IEnumerable<string>` should be enough.

Reply:
ReadOnlyCollection (next line) requires `IList<T>`